### ISSUE TITLE
FPS improvement?

### ISSUE THREAD

**Issue by Undetected0819** [133spider/AQLite#59]
Hi, i really love the work you've done, many useful and convenient features.

But, is there anyway to improve fps on high? I know it's because aqw itself is not optimized, but when i tried Grimoire(3.8+ ver) it seems to run smoother for some reason with the same settings(everything is default on aqlite's extra settings) even with animation on.
I also cleared the cache(Deleting the files inside the cache files in appdata method).

Though maybe it's on my end, i'm not really tech savy.

**Comment by 133spider** [comment]
Hey there, I've been brainstorming for a long time now to find a solution for FPS improvement in the launcher. 
I'm thinking about porting it over to C#, but I have to run thorough tests on C# before I can convert it over.

In the meantime, do you mind trying to change one setting in the launcher to see if you get a difference in performance?
Go to AQLite/resources/app/public/index.html and change all instances of "direct" to "window"
![image](https://user-images.githubusercontent.com/11169157/86065140-ba461c00-ba23-11ea-9f32-c6ab307d18ce.png)

**Comment by Undetected0819** [comment]
Sorry for the late reply, 
It seems to give a slight fps improvement, around 2 to 3 fps max, but still the same.
Tested it on Aranx, Doomwood infinite room and others with 3 mobs using VHL, SSG and IE, both with "direct" and "window".

I just noticed there was an update, my bad, and many thanks.

**Comment by 133spider** [comment]
Yeah this whole FPS issue is going to be a never-ending battle for me,
I won't be giving up anytime soon.
